packages/pds/src/bin.ts

```diff
@@ -45,8 +45,8 @@ const run = async () => {
     blobstore = new MemoryBlobStore()
   }
 
-  const { listener } = server(db, blobstore, keypair, cfg)
-  listener.on('listening', () => {
+  const pds = PDS.create({ db, blobstore, keypair, cfg })
+  pds.server.on('listening', () => {
```

Comment:
Any thoughts on making `PDS.create` async & only returning after this is listening?

Reply:
Listening is just such an effectful thing— I think it can be useful to have a way to create the server without listening, mostly for testing or using it "headlessly" as a library rather than a web service.  To take an example, in hapi there's `initialize()` which revs everything up and ensures the server is complete/healthy, and then `start()` which does all of that but additionally listens.  Drawing that analogy roughly over to the pds, you might have something like:

E.g.
```js
const app = PDS.create(/* ... */) // Creates app inertly
await app.initialize() // Ensures db connections look good, message queue might start processing
const { db, services } = app
await services(db).actor().getUser('x') // This works now
await app.start() // App is accepting traffic over http, could also call without initialize() first
```

Reply:
ah nice i quite like the sound of that 👌